src/components/AboutPage.js

```diff
@@ -59,74 +62,87 @@ const AboutPage = () => (
           <li>...</li>
           <li>...</li> */}
         </ul>
+        <p>
+          Contact us if you would like to get involved.
```

Comment:
Could you change this to something like: Contact us if you would like to get involved. We are always looking to partner with organizations who are committed to being anti-racist and want to help work for justice and equity through tech. You can also join our slack workplace [include link] to learn more from the organizing team.